examples/example.go

```diff
@@ -56,7 +73,7 @@ func (e *Examples) GetAssetById() (uipath.Asset, error) {
 
 	aHandler := uipath.AssetHandler{
 		Client:   e.Client,
-		FolderId: uint(folderID),
+		FolderId: folderID,
```

Comment:
The folderID appears to be an int, does that mean I don't have to cast it to uint?

Reply:
you can cast on the variable initalization